clang/include/clang/Basic/ParsedAttrInfo.h

```diff
@@ -165,4 +166,8 @@ const std::list<std::unique_ptr<ParsedAttrInfo>> &getAttributePluginInstances();
 
 } // namespace clang
 
+namespace llvm {
+extern template class CLANG_TEMPLATE_ABI llvm::Registry<clang::ParsedAttrInfo>;
```

Comment:
```suggestion
extern template class CLANG_TEMPLATE_ABI Registry<clang::ParsedAttrInfo>;
```

This is already in the `llvm` namespace.